### PR TITLE
 Beginnings of the login form. 

### PR DESCRIPTION
**Pull request by joeldenning**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -3,5 +3,6 @@
     "@babel/preset-env",
     "@babel/preset-react",
     "@babel/preset-typescript",
+    "@emotion/babel-preset-css-prop",
```
Comment (joeldenning) [inline .babelrc]: Emotion is a library for css in react. See https://emotion.sh/docs/introduction

File: src/login/login.resource.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export function performLogin(username, password) {
+  const token = window.btoa(`${username}:${password}`);
+  return window
+    .fetch(`/openmrs/ws/rest/v1/session`, {
```
Comment (joeldenning) [inline src/login/login.resource.tsx]: Making the API call to login to openmrs with the REST api

Comment (fatmali) [reply src/login/login.resource.tsx]: @joeldenning is it possible to abstract this into a separate file?

Comment (dkayiwa) [reply src/login/login.resource.tsx]: Will this work when the context is named anything other than 'openmrs'?

Comment (joeldenning) [reply src/login/login.resource.tsx]: @fatmali it already is in a separate file - `login.resource.tsx`

@dkayiwa no it won't, good catch. I'll change that.

Comment (joeldenning) [reply src/login/login.resource.tsx]: Going to merge this as-is and capture the need to account for other context's besides openmrs in https://github.com/openmrs/openmrs-esm-login/issues/2

**Comment by angshu** [comment]
LGTM to merge.

**Pull request merged**